docs/docs/0.7.2/documentation/security.md

```diff
@@ -1,8 +1,22 @@
 # Security
 
-Kuma helps you secure your existing infrastructure with mTLS. The following sections cover details of how it works.
+There are several communication use cases that can be secured when using Kuma.
 
-## Certificates
+## Data plane to data plane communication
+
+Kuma helps you secure your existing infrastructure with mTLS.
+
+### Mutual TLS
+
+Once a data plane has proved its identity to the control plane, it will be allowed to fetch its own identity certificate and a root CA certificate of the mesh.
+When establishing a connection between two data planes each side validates each other data plane certificate confirming the identity using the root CA of the mesh.
+
+mTLS is _not_ enabled by default. To enable it, apply proper settings in [Mesh](../../policies/mesh) policy.
+Additionally, when running on Universal you have to ensure that every data plane in the mesh has been configured with a Dataplane Token.
```

Comment:
```suggestion
Additionally, when running on Universal you have to ensure that every data plane in the mesh has been configured with a data plane Token.
```

Reply:
I think it should be either `data plane token` or `Dataplane Token`, because it is our termⓇ™️

Reply:
@jakubdyszkiewicz data plane proxy token